src/quo2/foundations/colors.cljs

```diff
@@ -193,6 +193,7 @@
    :beige     {50 "#CAAE93"
                60 "#AA927C"}})
 
+
```

Comment:
extra space

Reply:
Extra line is still there, though marked as resolved